tasks/config/run.js

```diff
@@ -269,7 +269,6 @@ module.exports = function (grunt) {
         '--config', 'test/plugin_functional/config.js',
         '--bail',
         '--debug',
-        '--kibana-install-dir', KIBANA_INSTALL_DIR,
```

Comment:
There is a problem in the functional tests that causes some code to be bundled twice: https://github.com/elastic/kibana/issues/43087

In this case it fails the test because there are two `angular` imports.

I checked with @spalger and this is the way we came up with to fix the problem.